sentry_sdk/utils.py

```diff
@@ -54,6 +54,8 @@
         Union,
     )
 
+    from gevent.hub import Hub
```

Comment:
Since `gevent` is an optional dependency, won't this break type checking for folks that don't have it installed? Would a conditional import help in any way?

Reply:
There's no conditional imports for typing unfortunately. So yes, if people leave `follow_imports` at `normal` for `sentry_sdk` they will see an error if they don't have `types-gevent` installed, but the same is true for any of the other typing only depdencies that are only listed in `dev-requirements.txt`, so this would be nothing new.

Generally the expectation with type hints is that people are responsible for managing `typing-` requirements and otherwise silence the errors by setting `follow_imports` to `silent` or `skip`.